tensorflow/python/pywrap_tensorflow.py

```diff
@@ -46,8 +46,12 @@
     sys.setdlopenflags(_default_dlopen_flags)
 except ImportError:
   msg = """%s\n\nFailed to load the native TensorFlow runtime.\n
-See https://github.com/tensorflow/tensorflow/blob/master/tensorflow/g3doc/get_started/os_setup.md#import_error\n
-for some common reasons and solutions.  Include the entire stack trace
+Visit https://github.com/tensorflow/tensorflow/blob/master/tensorflow/docs_src/install/index.md\n
+and click on the link that corresponds to your operating system to read about\n
```

Comment:
There is no need for this line break here, because it is the middle of a sentence. With this line break, the sentence would be broken into two lines separated by an empty line.
Same below.